great_expectations/self_check/util.py

```diff
@@ -1366,13 +1366,14 @@ def candidate_test_is_on_temporary_notimplemented_list_cfe(context, expectation_
 
 def build_test_backends_list(
     include_pandas=True,
-    include_spark=True,
+    include_spark=False,
```

Comment:
Keeping this signature consistent. Only `pandas`, `sqlalchemy` and `sqlite` are `True` by default